src/components/card/card.style.ts

```diff
@@ -74,8 +83,84 @@ const StyledCard = styled.div<StyledCardProps>`
   `}
 `;
 
+interface StyledCardContentProps
+  extends Pick<CardContextProps, "roundness" | "spacing"> {
+  interactive?: boolean;
+  href?: string;
+  hasFooter: boolean;
+  target?: string;
+  rel?: string;
+}
+
+const StyledCardContent = styled.div.attrs(
+  ({ href, onClick }: Pick<StyledCardProps, "href" | "onClick">) => {
+    if (href) {
+      return { as: "a" };
+    }
+
+    if (onClick) {
+      return { as: "button", role: "button", type: "button" };
+    }
+
+    return {};
+  }
+)<StyledCardContentProps>`
+  ${({ interactive, theme }) =>
+    interactive &&
+    css`
+      cursor: pointer;
+
+      ${!theme.focusRedesignOptOut &&
+      css`
+        :focus {
+          ${addFocusStyling()};
+          position: relative;
+        }
+      `}
+    `}
+
+  outline: none;
+  text-decoration: none;
+  background-color: inherit;
+  border: none;
+  padding: 0;
+
+  ${({ spacing }) => `
+    padding: ${paddingSizes[spacing]};
+    margin: ${marginSizes[spacing]};
+  `}
```

Comment:
**suggestion**: it appears as `StyledCard` already provides padding on line 61, removing the negative margins and padding achieves the same positioning as with it.

```suggestion
```

Reply:
Unfortunately that doesn't work as you lose the cursor and focus styling when you don't have the above. I also tried removing the styling on line 61 but it causes the footer to look misaligned.

![image](https://github.com/Sage/carbon/assets/44157880/c39179af-9855-479c-8a53-126d9ce59199)